airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/VersionCell.tsx

```diff
@@ -111,7 +111,11 @@ const VersionCell: React.FC<IProps> = ({
             <LoadingButton
               isLoading={isSubmitting}
               type="submit"
-              disabled={(isSubmitting || !dirty) && currentVersion === version}
+              disabled={
+                (isSubmitting || !dirty) &&
+                currentVersion === version &&
```

Comment:
Maybe it's better to add a separate variable like `isConnectorUpdatable`? So it will be more obvious why button is disabled